src/agl/web_runtime_agl.cc

```diff
@@ -339,6 +339,8 @@ bool WebAppLauncherRuntime::Init() {
       role_ = "homescreen";
     else if (id_.rfind("webapps-homescreen", 0) == 0)
       role_ = "homescreen";
+    else if (id_.rfind("webapps-html5-background", 0) == 0)
+      role_ = "background";
```

Comment:
These roles will go away, but I guess for the time being we have thee de-couple when they should be one and the same.

Reply:
I've just checked and role_ is not used anymore. So I will just provide a patch to drop this part.

Reported as https://jira.automotivelinux.org/browse/SPEC-4282